components/AppHeader/AppHeader.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+
+export default function AppHeader(){
+return(
+    <Header>
+        <p>MatchPal</p>
```

Comment:
Maybe you want to add change the `p` to an `h` tag here, since it is a heading, right? Just for screenreader purposes. Also you can change that later I guess.

Reply:
yes, thanks for the suggestion. I will change that later on, because I am still thinking of building a fancy logo and replacing the text with it haha

Reply:
Aaah I see that, as placeholder it is just perfect 😄